pkgs/applications/audio/muse/default.nix

```diff
@@ -45,13 +47,13 @@ stdenv.mkDerivation {
 
   nativeBuildInputs = [
     pkgconfig
-    gitAndTools.gitFull
   ];
 
   buildInputs = [
     libjack2
-    qt5.qtsvg
-    qt5.qttools
+    wrapQtAppsHook
+    qtsvg
+    qttools
     cmake
```

Comment:
`cmake` and `wrapQtAppsHook` and possibly `qttools` belong to `nativeBuildInputs`.

Reply:
Moved.